### PR TITLE
Fix #2059: Remove inaccuracy in page number of existing teams on participate tab

### LINKED ISSUE #2059: Inaccuracy in page number in Existing Teams
### Observed Behaviour
When we have less than 10 teams on existing teams page, the intial page count is shown as 0. (Refer to the screenshot) 

### Expected Behaviour
The initial count should be 1.

### Steps to Reproduce
Go to any challenge in which you have not particpated. Select the `Participate` tab.

### Deliverables
- [ ] Update the intial page number on the participant side.
- [ ] Check for any other pagination issue and fix that too.

@deshraj @RishabhJain2018 

capture

### PR DESCRIPTION
**Pull request by Sanji515**
Fix #2059 Now I've added two cases - First for the default, when `challenge.existTeam.results` is empty (i.e. it has no team name to be displayed) and another when `challenge.existTeam.results` is not empty (means it has team names to be displayed).

### CODE REVIEW

**Comment by vkartik97** [comment]
Hi @Sanji515 
If Travis CI is not passing because of `test_get_remaining_submission_when_submission_made_three_days_back`, then no need to worry!
[EvalAI/pull#2063](https://github.com/Cloud-CV/EvalAI/pull/2063) deals with this issue.
Please let me know if you have questions. :)

**Comment by Sanji515** [comment]
@vkartik97 What should I do now?

**Comment by vkartik97** [comment]
@Sanji515 
Your PR deals with Frontend, which has nothing to do with the Failing Tests.
So, No need to worry, the Travis CI's issue will be resolved soon :)

**Comment by Sanji515** [comment]
@vkartik97 okay thanks!!

File: frontend/src/views/web/challenge/participate.html [hunk 1]
```diff
@@ -28,7 +28,11 @@
                                                 <a ng-class="challenge.isPrev" class="btn-floating btn-pagination waves-effect waves-light " ng-click="challenge.load(challenge.existTeam.previous)">
                                                     <i class="fa fa-chevron-left"></i>
                                                 </a>
-                                                <span class="pagination-title"> <strong class="text-med-black"> Page {{challenge.currentPage | ceil}} of {{challenge.existTeam.count/10 | ceil}} </strong></span>
+                                                <span class="pagination-title">
+                                                    <strong class="text-med-black" ng-if="!challenge.existTeam.results"> Page 1 of {{challenge.existTeam.count/10 | ceil}} </strong>
```
Comment (gautamjajoo) [inline frontend/src/views/web/challenge/participate.html]: I think so you shouldn't hardcore this here, you could make changes in the `js` file.

Comment (Sanji515) [reply frontend/src/views/web/challenge/participate.html]: @gautamjajoo done.

**Comment by Sanji515** [comment]
@gautamjajoo @RishabhJain2018 I've done the changes as mentioned above, please review.

**Comment by RishabhJain2018** [comment]
@gautamjajoo Can you please pull and test on your local.

**Comment by gautamjajoo** [comment]
Hey @Sanji515 . It works when there are 2-3 participant teams but if you create around 20 teams, you could see that the page number as `Page 0 of 3`. Check the pagination limit of the page and fix according to it?

**Comment by Sanji515** [comment]
> Hey @Sanji515 . It works when there are 2-3 participant teams but if you create around 20 teams, you could see that the page number as `Page 0 of 3`. Check the pagination limit of the page and fix according to it?

@gautamjajoo I'm unable to reproduce the bug, on my side it's working fine, can you please show me exactly when it's happening.

**Comment by gautamjajoo** [comment]
@Sanji515 Create around 20 teams and you could see the bug.

**Comment by Sanji515** [comment]
@gautamjajoo I've checked that too but still can't see that bug

**Comment by Sanji515** [comment]
@gautamjajoo Changed the previous implementation & added the default message when there is no team exists, please review.

**Comment by gautamjajoo** [comment]
Hey! I have created around 25 teams and the pagination is not as expected. Please see the attached screenshot.
screenshot from 2019-02-12 15-28-57

**Comment by Sanji515** [comment]
@gautamjajoo have you checked this on my last commit [default msg when there is no team created](/Cloud-CV/EvalAI/pull/2065/commits/0f91ddfb70c7ec8a1b163c68bdf87b071e8e06a5)?

**Comment by Sanji515** [comment]
@RishabhJain2018 Please review

**Pull request merged**